xml/System/Type.xml

```diff
@@ -13032,9 +13039,17 @@ Types that are defined in the .NET Standard are not marked with <xref:System.Ser
         <ReturnType>System.Boolean</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
-        <remarks>To be added.</remarks>
+        <summary>Gets a value indicating whether the <see cref="T:System.Type" /> is a signature type.</summary>
+        <value><see langword="true" /> if the <see cref="T:System.Type" /> is a signature type; otherwise, <see langword="false" />.</value>
```

Comment:
```suggestion
        <value>
          <see langword="true" /> if the <see cref="T:System.Type" /> is a signature type; otherwise, <see langword="false" />.</value>
```